proxy/proxy.go

```
@@ -1253,6 +1263,7 @@ func (p *Proxy) errorResponse(ctx *context, err error) {
 	}
 
 	switch {
+	case code == 404:
```

What is the intent here?

---

I think to ignore the error log, which is fine in this case.
Do you have another opinion?

---

I see this is coming from fastcgi special case when script is not found. I think it should be handled inside fastcgi roundtripper internally if it is not really a proxy error.

---

Yeah great idea!

---

done ! thanks.